lib/burp/html/issue.rb

```diff
@@ -84,6 +84,19 @@ def method_missing(method, *args)
       method_names = translations_table.fetch(method, method.to_s)
       method_names = [method_names].flatten
 
+      # Process the Location field
+      if method.to_s == 'location'
+        location = @html.at_xpath('//span[contains(@class, "BODH1")]')&.text
```

Comment:
Aren't we dealing with a single Issue at a time? Why do we need '//'?

Reply:
We don't need it but we've used that same xpath across the plugin. I've replaced this one with a more specific search vs searching the whole document.